### ISSUE TITLE
About install 

### ISSUE THREAD

**Issue by Rickyzhang1990** [warrenlr/LINKS#9]
Hi Rene
Thank you for your tools,but I have a problem in install LINKS.
when I run LINKS ,the error is as follows:
Can't locate Time/HiRes.pm in @INC (@INC contains: /lustre/work/chaozhang/soft/loadfile/links_v1.8.5/./lib/bloomfilter/swig /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at ./LINKS line 28.
BEGIN failed--compilation aborted at ./LINKS line 28.
 but I do have installed the packages before. this is my question1.
The other is when i am running this command,the error is 
perl LINKS.pl 
perl: symbol lookup error: /lustre/work/chaozhang/soft/loadfile/links_v1.8.5/./lib/bloomfilter/swig/BloomFilter.so: undefined symbol: Perl_Gthr_key_ptr

so which is the major reason?

great thanks 
chaozhang

**Comment by warrenlr** [comment]
Hi chaozhang,

You may have to recompile the Bloom Filter library with your current PERL version.
(error is likely due to incompatibility between the pre-built perl module and your version of PERL)

I recommend you follow the build/install instructions in the README.

let me know how it goes.  All the best,
Rene

**Comment by Rickyzhang1990** [comment]
thank you for your reply , I will try and then tell you the result

**Comment by Rickyzhang1990** [comment]
hi Rene, I follow the Readme ,and the the output of 
$./test.pl like this ,
de novo bf tests done 
Storing filter. Filter is 125000000bytes.
Loading header...
premade bf tests done
Filter Info: Pop - 20, numHash - 5, kmerSize - 20, size - 1000000000
0 TAGAA found
1 AGAAT found
2 GAATC found
3 AATCA found
4 ATCAC found
5 TCACC found
6 CACCC found
7 ACCCA found
8 CCCAA found
9 CCAAA found
10 CAAAG found
11 AAAGA found
Done!
It seems like a normal result .and the forth step of install I did not change the path.
but when I am running the LINKS ,the error raised as I described before .
Thank you for your advice
